p2p/broadcaster.go

```diff
@@ -70,17 +76,22 @@ func NewBroadcaster(args ArgsBroadcaster) (*broadcaster, error) {
 			counter:      uint64(time.Now().UnixNano()),
 			privateKey:   args.PrivateKey,
 		},
-		clients:       make([]core.BroadcastClient, 0),
-		joinTopicName: args.Name + joinTopicSuffix,
-		signTopicName: args.Name + signTopicSuffix,
+		clients:                make([]core.BroadcastClient, 0),
+		joinTopicName:          args.Name + joinTopicSuffix,
+		signTopicName:          args.Name + signTopicSuffix,
+		antifloodStatusHandler: args.AntifloodStatusHandler,
 	}
-
 	pk := b.privateKey.GeneratePublic()
 	b.publicKeyBytes, err = pk.ToByteArray()
 	if err != nil {
 		return nil, err
 	}
 
+	b.relayerMessageHandler.antifloodComponents, err = b.createAntifloodComponents(args.AntifloodConfig)
```

Comment:
I would have injected all the antiflood components instead of creating them here. 
The creation should have been done in the ethElrondBridgeComponents struct